WcaOnRails/db/migrate/20180701160042_add_total_number_of_rounds_to_round.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class AddTotalNumberOfRoundsToRound < ActiveRecord::Migration[5.2]
+  def up
+    add_column :rounds, :total_number_of_rounds, :integer, null: false
+
+    # Round types has changed over time.
```

Comment:
nit: I think this comment is vestigial now, because we're not storing round types in the database, just the total number of rounds.